src/media/js/site/components/header.js

```diff
@@ -40,23 +66,34 @@ class HeaderLogin extends React.Component {
     logoutHandler: React.PropTypes.func.isRequired,
     isLoggedIn: React.PropTypes.bool.isRequired,
   };
+
+  renderAnonNav() {
+    return (
+      <nav className="header--nav header--login-nav">
```

Comment:
At one point in time I styled the different <nav> elements differently, but I realize now that I actually managed to do away with that. The second className isn't needed at all.